mlir/test/Integration/Dialect/SparseTensor/GPU/CUDA/dump-ptx.mlir

```diff
@@ -1,5 +1,5 @@
 // RUN: mlir-opt %s \
-// RUN:  | mlir-opt -gpu-lower-to-nvvm-pipeline -debug-only=serialize-to-isa \
+// RUN:  | mlir-opt -gpu-lower-to-nvvm-pipeline -debug-only=serialize-to-isa -debug-only=dump-sass \
```

Comment:
Missing check lines?

Reply:
It's hard to check SASS as llvm doesn't have control. But I added a check that checks most common instructions.

Reply:
Yes I meant just checking that the flow is working, so even a SASS header or something is enough, thanks!